Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
```

Comment:
We don't? I presumed from the comments elsewhere that it was necessary to kill ` postgres ` to migrate databases?